pages/api/auth/[...nextauth].js

```diff
@@ -17,6 +17,11 @@ export default NextAuth({
       clientId: process.env.TWITTER_ID,
       clientSecret: process.env.TWITTER_SECRET
     }),
+    Providers.Okta({
+      clientId: process.env.OKTA_ID,
+      clientSecret: process.env.OKTA_SECRET,
+      domain: process.env.OKTA_DOMAIN
+    }),
```

Comment:
For now, I don't wish we start adding every provider to the dev app, it's totally fine if you do it for local development though, and I encourage you to actually test your changes locally (which I guess you did since it was added here)! Could you please remove this section from this PR?
```suggestion
```